### PR TITLE
Cypress/E2E: Fix MM-T924 by creating enough users to page through

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix MM-T924 by creating enough users to page through

#### Ticket Link
failed on daily Cypress unstable test run

### CODE REVIEW

File: e2e/cypress/support/api/helpers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// See LICENSE.txt for license information.
+
+export function buildQueryString(queryParams = {}) {
+    let queryString = '';
+    Object.entries(queryParams).forEach(([k, v], index) => {
+        if (index > 0) {
+            queryString += '&';
+        }
+
+        queryString += `${k}=${encodeURIComponent(v)}`;
```
Comment (josephbaylon) [inline e2e/cypress/support/api/helpers.js:11]: Shouldn't key also be encoded?

**Review by josephbaylon — APPROVED** [review]
LGTM. Just a minor question.

File: e2e/cypress/support/api/helpers.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/support/api/helpers.js:11]: I don't think so and honestly don't know if we're doing it in other place. In mattermost-redux, we're only encoding the value as well.  Key is something we have control/validation in server and I don't think we allow a key that is unescaped or will require to be encoded. But happy to change if necessary.

**Review by furqanmlk — APPROVED** [review]
All related test cases passed ✅ 
Looks good !

**Pull request merged**